cl/lib/elasticsearch_utils.py

```diff
@@ -751,16 +754,34 @@ def build_custom_function_score_for_date(
     results based on a child document field.
     :param default_score: The default score to return when the document lacks
     the sort field.
+    :param default_current_date: The default current date to use for computing
+     a stable date score across pagination in the V4 Search API.
     :return: The modified QueryString object with applied function score.
     """
 
+    midnight_current_date = (
+        datetime.datetime.combine(default_current_date, datetime.time())
+        if default_current_date
+        else None
+    )
+    default_current_time = (
+        int(midnight_current_date.timestamp() * 1000)
+        if midnight_current_date
+        else None
+    )
```

Comment:
I think we can simplify this logic by using a single if statement

```suggestion
    midnight_current_date = None
    default_current_time = None
    if default_current_date:
        midnight_current_date = datetime.datetime.combine(
            default_current_date, datetime.time()
        )
        default_current_time = int(midnight_current_date.timestamp() * 1000)
```

Reply:
changed, thanks for the suggestion!